stellarphot/photometry/tests/test_profiles.py

```diff
@@ -102,3 +112,32 @@ def test_radial_profile_exposure_is_nan():
     assert np.isnan(rad_prof.noise(c, np.nan)[-1])
     assert all(np.isfinite(rad_prof.curve_of_growth.profile))
     assert all(np.isfinite(rad_prof.radial_profile.profile))
+
+
+def test_radial_profile_with_background():
+    # Regression test for #328 -- image with a background level
```

Comment:
Again, I believe the logic here is sound, though it took a bit for me to understand it.